godtools/GodToolsConstants.swift

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
I also made a file for constants, but, instead of making a class I made a `struct`, what do you think? Also, the file I created is called `GTConstants`; I'm adding the prefix `GT` to every file, can we refactor and add the prefix `GT` to every file?